v2/internal/frontend/desktop/windows/win32/window.go

```diff
@@ -37,6 +38,16 @@ func IsWindowMaximised(hwnd uintptr) bool {
 	return style&WS_MAXIMIZE != 0
 }
 
+func IsWindowMinimized(hwnd uintptr) bool {
+	style := uint32(getWindowLong(hwnd, GWL_STYLE))
+	return style&WS_MINIMIZE != 0
+}
+
+func IsWindowNormal(hwnd uintptr) bool {
+	style := uint32(getWindowLong(hwnd, GWL_STYLE))
```

Comment:
Could we do `!IsWindowMinimised() && !IsWindowMaximised()` ?

Reply:
is changed